target/linux/ramips/dts/ADSLR-G7.dts

```diff
@@ -26,28 +26,27 @@
 		bootargs = "console=ttyS0,57600";
 	};
 
-	gpio-leds {
+	leds {
 		compatible = "gpio-leds";
 
-
-		led_wan_blue: wan_blue {
-			label = "adslr-g7:blue:wan";
+ 		led_sys: sys {
+			label = "g7:blue:sys";
 			gpios = <&gpio0 7 GPIO_ACTIVE_HIGH>;
 		};
-
 	};
 
-	gpio-keys-polled {
-		compatible = "gpio-keys-polled";
-		poll-interval = <20>;
+	
+	keys {
+		compatible = "gpio-keys";
 
-		reset {
+ 		reset {
```

Comment:
Thanks for taking care of the changes so far.
This line seems to contain a space at the beginning.

Reply:
Space removed